### ISSUE TITLE
Switch background color by user’s input

### ISSUE THREAD

**Issue by masakudamatsu** [masakudamatsu/luminance-picker#40]
If the user enters a color whose luminance is above the threshold (sqrt(21) = 4.5826), switch the background color to dark grey (1:3 contrast ratio to the threshold). Otherwise, it’s off-white (3:1 to the threshold).

The user enters a color code (or RGB values)

Convert the input color code into its contrast ratio

If the contrast ratio is above the threshold, turn on the `darkMode` state.

Use `props.darkMode` in styled-components to turn the background color into dark grey. 

Animate background color change with `opacity` property
- see https://openclassrooms.com/en/courses/5625816-create-modern-css-animations/5995831-create-more-peformant-color-animations-using-the-css-opacity-property (log in with Google account)

**Comment by masakudamatsu** [comment]
We use `rgb(117, 117, 117)` as the threshold luminance level
- Its contrast ratio, 4.55:1, is the closest to sqrt(21) = 4.5826.
- Its contrast ratio to pure white is 1 to 4.6.
- cf. `rgb(118, 118, 118)` has a contrast ratio of 4.62:1 to pure black, 1:4.54 to pure white.
<img width="923" alt="Screenshot 2020-09-09 at 13 25 23" src="https://user-images.githubusercontent.com/36348559/92554652-d0345080-f2a0-11ea-8105-18d3af18a70c.png">
<img width="923" alt="Screenshot 2020-09-09 at 13 25 14" src="https://user-images.githubusercontent.com/36348559/92554660-d32f4100-f2a0-11ea-94c3-d330219d1ee6.png">
